### PR TITLE
[MIG][15.0] account_invoicing_mode_at_shipping (renamed)

### PR DESCRIPTION
**Pull request by TDu**
This module migration depends on #1243 

It also takes over this pr #1101

### CODE REVIEW

**Comment by rousseldenis** [comment]
@TDu To easy review, could you add the dependency PR link in this description ? Thanks

**Comment by StefanRijnhart** [comment]
Can you remove the migration script here as well?

**Comment by TDu** [comment]
@StefanRijnhart Done

File: partner_invoicing_mode_at_shipping/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import models
+from . import tests
```
Comment (StefanRijnhart) [inline partner_invoicing_mode_at_shipping/__init__.py]: Please remove this import 
```
************* Module partner_invoicing_mode_at_shipping
partner_invoicing_mode_at_shipping/__init__.py:2: [E7903(test-folder-imported), ] Test folder imported in module partner_invoicing_mode_at_shipping

Check for Odoo modules using pylint......................................Failed
- hook id: pylint_odoo
- exit code: 2

************* Module partner_invoicing_mode_at_shipping
partner_invoicing_mode_at_shipping/__init__.py:2: [E7903(test-folder-imported), ] Test folder imported in module partner_invoicing_mode_at_shipping
```

**Review by StefanRijnhart — APPROVED** [review]

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
@pedrobaeza this one can either be merge :) thanks !

**Comment by pedrobaeza** [comment]
It can't be merged without removing the unreleased dependency

**Comment by houssine78** [comment]
@pedrobaeza the dependency is on the account_invoice_base_invoicing_mode which has been merged yesterday.

**Comment by StefanRijnhart** [comment]
@houssine78 `test-requirements.txt` needs to be removed by the author @TDu.

**Comment by TDu** [comment]
@StefanRijnhart I removed the `test-requirements.txt` change, should be good to merge.

**Comment by StefanRijnhart** [comment]
Thanks!
/ocabot merge nobump

**Pull request merged**